lms/static/scripts/frontend_apps/components/LaunchErrorDialog.js

```diff
@@ -161,21 +160,52 @@ export default function LaunchErrorDialog({
           </p>
         </ErrorModal>
       );
+    case 'd2l_group_set_not_found':
+      return (
+        <ErrorModal
+          busy={busy}
+          error={error}
+          title="Assignment's group category no longer exists"
+        >
+          <p>
+            Hypothesis couldn&apos;t load this assignment because the
+            assignment&apos;s group category no longer exists.
+          </p>
+          <p>
+            <b>
+              To fix this problem, an instructor needs to edit the assignment
+              settings and select a new group category.
+            </b>
+          </p>
+        </ErrorModal>
+      );
 
     case 'blackboard_group_set_empty':
     case 'canvas_group_set_empty':
-    case 'd2l_group_set_empty':
       return (
         <ErrorModal
           busy={busy}
           error={error}
           title="Assignment's group set is empty"
         >
           <p>The group set for this Hypothesis assignment is empty. </p>
+          <p>
+            <b>different group set for this assignment.</b>
```

Comment:
This sentence is incomplete.

Reply:
Oh :facepalm:, I had tunnel vision for this I only was reading the set vs category parts.